packages/bundler-plugin-core/src/types.ts

```diff
@@ -241,9 +241,26 @@ export interface Options {
      * Defaults to `false`.
      */
     injectBuildInformation?: boolean;
+
+    /**
+     * Metadata associated with this module.
+     * The metadata is serialized and can be looked up at runtime by URL.
+     *
+     * Note: This option is currently only supported by `@sentry/webpack-plugin`.
+     */
+    moduleMetadata?: object | ModuleMetadataCallback;
   };
 }
 
+export interface ModuleMetadataCallbackArgs {
+  org?: string;
+  project?: string;
+  release?: string;
+  dist?: string;
```

Comment:
I don't think we need dist as callback arg since it is never generated by the plugin itself.